capstone/src/components/User/Adventurer/FindAdventure/FindAdventure.jsx

```diff
@@ -0,0 +1,34 @@
+import './FindAdventure.css';
+import { useState } from 'react';
+import Map from './Map/Map';
+
+export default function FindAdventure() {
+  const [displayExperience, setDisplayExperience] = useState('map');
+  return (
+    <div className="findAdventure">
+      <h2>Find your next adventure:</h2>
+      <ExperienceDisplayer
+        displayExperience={displayExperience}
+        setDisplayExperience={setDisplayExperience}
+      />
+      <Map />
+    </div>
+  );
+}
+
+export function ExperienceDisplayer({ displayExperience, setDisplayExperience }) {
+  if (displayExperience === 'map') {
+    return (
+      <div className="experienceDisplayer">
+        <button type="button" value="map" className="chosenDisplay" disabled>Map</button>
+        <button type="button" value="list" className="notChosenDisplay" onClick={(e) => setDisplayExperience(e.target.value)}>List</button>
```

Comment:
Is .chosenDisplay and .notChosenDisplay defined somewhere else? I couldn't find them in this PR.

Reply:
Oh, I only included the className but I haven't work on the CSS 😅